packages/tools/src/utilities/math/polyline/projectTo2D.ts

```diff
@@ -0,0 +1,48 @@
+import { utilities } from '@cornerstonejs/core';
+import type { Types } from '@cornerstonejs/core';
+
+/**
+ * Projects a polyline from 3D to 2D by reducing one dimension.
+ *
+ * @param polyline - The polyline to be projected.
+ * @returns An object containing the shared dimension index and the projected polyline in 2D.
+ * @throws Error if a shared dimension index cannot be found for the polyline.
+ */
+export function projectTo2D(polyline: Types.Point3[]) {
+  // We need to reduce one dimension to 2D, so basically
+  // we need to find the dimension index that is shared by all points
+  // Use the first three points, two is enough but three is more robust
+  let sharedDimensionIndex;
+
+  const testPoints = utilities.getRandomSampleFromArray(
+    polyline,
+    Math.min(50, polyline.length)
+  );
+  for (let i = 0; i < 3; i++) {
```

Comment:
I didn't get why `getRandomSampleFromArray` is called with `size` set to 50 but you are using only 3. Why not get only 3 points instead of 50?